int8 launchStack
---
string message